core/templates/dev/head/app.js

```diff
@@ -80,7 +80,7 @@ oppia.constant('COMPONENT_NAME_DEFAULT_OUTCOME', 'default_outcome');
 oppia.constant('CURRENT_ACTION_SCHEMA_VERSION', 1);
 oppia.constant('CURRENT_ISSUE_SCHEMA_VERSION', 1);
 oppia.constant('EARLY_QUIT_THRESHOLD_IN_SECS', 45);
-oppia.constant('NUM_INCORRECT_ANSWERS_THRESHOLD', 5);
+oppia.constant('NUM_INCORRECT_ANSWERS_THRESHOLD', 3);
```

Comment:
Not sure if this would be too small a value or not (its experimental anyway). So, I'm okay with having 3 for now, but we should be prepared to change it to 5 also.